lib/core/options.js

```diff
@@ -268,6 +268,23 @@ $.extend(RESUtils.options, {
 		}
 		RESStorage.setItem('RESoptions.' + moduleID, JSON.stringify(minify(modules[moduleID].options)));
 	},
+	loadOptions: function(moduleID) {
+		var needsStorage = false;
+		Object.getOwnPropertyNames(modules[moduleID].options || {}).some(function(optionKey) {
+			var type = modules[moduleID].options[optionKey].type;
+			if (type !== 'button') {
+				needsStorage = true;
+				return true;
+			}
```

Comment:
This can be simplified a bit:

	var needsStorage = Object.getOwnPropertyNames(modules[moduleID].options || {}).some(function(optionKey) {
		var type = modules[moduleID].options[optionKey].type;
		return type !== 'button';
	});